.github/actions/run-api-tests/script.sh

```diff
@@ -7,9 +7,9 @@ fi
 
 export ENV_PATH="$(pwd)/testApp/.env"
 export JWT_SECRET="aSecret"
-
+export NODE_OPTIONS="--max_old_space_size=6144"
```

Comment:
Maybe we should add a comment why we need this configuration.

Reply:
let me do a test without the export and see if it still works if it  could work without this it would be better